specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json

```diff
@@ -0,0 +1,1972 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2.1-preview.1",
```

Comment:
Should this be 2.1-preview.2 based on the title?

Reply:
Yes, it should. Thanks!